source/code-snippets/aggregation/agg.js

```diff
@@ -0,0 +1,48 @@
+const { MongoClient } = require('mongodb');
+const uri = process.env.DRIVER_REF_URI;
+const client = new MongoClient(uri);
+
+async function run() {
+    try {
+        await client.connect();
+
+        // begin data insertion
+        const db = client.db("aggregation");
+        const coll = db.collection("restaurants");
+
+        const docs = [
+            { "stars": 3, "categories": ["Bakery", "Sandwiches"], "name": "Rising Sun Bakery" },
```

Comment:
Do you have a formatter set up that you could run this code through? It's not wrong by its odd to see quotations around keys that could be written like `{ stars: 3, categories: [] }`. I also notice there's a mix of single and double quotes, best to stick with one throughout.

If not maybe throw this code through: https://prettier.io/playground/ and change around the settings to your desired format.

Reply:
good catch! accepted